sleap_roots/lengths.py

```diff
@@ -107,7 +108,7 @@ def get_root_lengths_max(pts: np.ndarray) -> np.ndarray:
 def get_grav_index(
```

Comment:
Same here: only input traits in the trait definition can be arguments to the function.